app/managers/session_manager.ts

```diff
@@ -187,7 +187,7 @@ class SessionManager {
 
     private onSessionExpired = async (serverUrl: string) => {
         this.terminatingSessionUrl.add(serverUrl);
-        await logout(serverUrl, false, false, true);
+        await logout(serverUrl, undefined, true, false, true);
```

Comment:
I'm unsure why we call this, we changed `skipServerLogout` to true and `skipEvents` is true. We won't do anything in logout apart from return `{data: true};`? Unless changing `skipServerLogout` to true was a mistake?

Reply:
You are completely right. But it makes kind of sense :sweat_smile:

If the session expired, in theory it expired on the server, so we don't need to contact the server.
And if we are in this part of the code, is because the SESSION_EXPIRED event was triggered, so we don't have to trigger it again.

I guess we can remove the logout completely, yes, but I have mixed feelings because "logically" it makes sense to have it here.

Reply:
So I guess this call is here in case there are changes to the logout function in future and we forget to add the call back to `onSessionExpired`? Should we add a comment above this call in case someone else removes it?